src/fragments/lib/logging/common/remote_configuration/remote_configuration.mdx

```diff
@@ -0,0 +1,220 @@
+      runtime: lambda.Runtime.NODEJS_18_X,
+      code: lambda.Code.fromAsset(path.dirname(path.join(lambdaConfig))),
+      handler: "remotelogging.main",
+      environment: {
+        BUCKET: remoteConfigBucket.bucketName,
+        KEY: configFileName
+      }
+    })
+
+    remoteConfigBucket.grantRead(handler)
+
+    const api = new apigateway.RestApi(this, "AmplifyRemoteLogging-API", {
+      restApiName: "Logging API",
+      description: "API Gateway for Remote Logging"
+    })
+
+    const getRemoteLoggingIntegration = new apigateway.LambdaIntegration(handler)
+    const loggingConstraints = api.root.addResource('loggingconstraints')
+    const getLoggingConstraints = loggingConstraints.addMethod('GET', getRemoteLoggingIntegration, {
+      authorizationType: apigateway.AuthorizationType.IAM
+    })
+
+    const apiInvokePolicy = new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      resources: [ getLoggingConstraints.methodArn ],
+      actions: ['execute-api:Invoke']
+    })
+
+    authRole.addToPrincipalPolicy(apiInvokePolicy)
+    unAuthRole.addToPrincipalPolicy(apiInvokePolicy)
+
+  }
+}
+```
+
+## Sample Lambda handler
+Below is a sample lambda that reads and returns the `remoteloggingconstraints.json` from AWS S3.
+Note that caching and ETag is used.  To reduce data usage, if the logging levels has not changed since the last fetch, there is no need to return the log levels.
+
+```js
+const { S3Client, GetObjectCommand } = require("@aws-sdk/client-s3")
+const s3 = new S3Client({})
+const bucketName = process.env.BUCKET
+const key = process.env.KEY
+let cachedConfig = {
+    expiresOn: 0, 
+    ETag: '', 
+    config: ''
+}
+
+exports.main = async function(event, context) {
+  try {
+    if (event.httpMethod === "GET") {
+      if (!cachedConfig.config || Date.now() > cachedConfig.expiresOn) {
+        // refresh cache if cache is invalid
+        const command = new GetObjectCommand({ Bucket: bucketName, Key: key })
+        const s3Resp = await s3.send(command)
+        await setCachedConfig(s3Resp)
+      }
+
+      if (event.headers['If-None-Match'] === cachedConfig.ETag) {
+        // return 304 not modified if config has not changed
+        return {
+          statusCode: 304,
+        }
+      } else {
+        // return updated/modified config with latest ETag
+        return {
+          statusCode: 200,
+          headers: {"'If-None-Match'": cachedConfig.ETag },
+          body: cachedConfig.config
+        }
+      }
+    }
+  } catch(error) {
+    const resp = error.stack || JSON.stringify(error, null, 2)
+    return {
+      statusCode: 400,
+        headers: {},
+        body: JSON.stringify(resp)
+    }
+  }
+}
+
+const setCachedConfig = async (s3Resp) => {
+  cachedConfig = {
+    expiresOn: Date.now() + 600 * 1000, //10 minutes
+    ETag: s3Resp.ETag.replace(/\"/gi, ''), //remove \" from string
+    config: await s3Resp.Body.transformToString()
+  }
+}
+```
+
+## Sample `remoteloggingconstraints.json`
```

Comment:
Can we change the title to "Creating remote configuration file"

Reply:
header updated as suggested.